Tasks/PowerShellOnTargetMachines/task.json

```diff
@@ -11,9 +11,9 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
+        "Major": 2,
         "Minor": 0,
-        "Patch": 47
+        "Patch": 0
     },
     "minimumAgentVersion": "1.104.0",
```

Comment:
min agent version should be 2.* as v1 does not support PS3 handler. Better to keep this as latest released agent